src/main/java/minijava/parser/Parser.java

```diff
@@ -153,8 +160,9 @@ private void parseMethod() {
   /** Parameters -> Parameter | Parameter , Parameters */
   private void parseParameters() {
     parseParameter();
-    if (isCurrentTokenTypeOf(COMMA)) {
-      parseParameters();
+    while (isCurrentTokenTypeOf(COMMA)) {
+      expectAndConsume(COMMA);
+      parseParameter();
```

Comment:
I figured that the bugs I found and fixed demand some explanation.

This one is the one from the other PR: Although Parameterlists of length > 1 in method prototypes are checked for with `COMMA`, die actual `COMMA` token isn't consumed, so  `IDENT COMMA IDENT` would complain that `COMMA` isn't an `IDENT`.